crates/volta-core/src/tool/package/mod.rs

```diff
@@ -126,13 +126,28 @@ impl Display for Package {
 pub struct DirectInstall {
     staging: TempDir,
     manager: PackageManager,
+    name: Option<String>,
```

Comment:
If I've followed the writeup and the PR correctly, this is the *package* name for the installation? Could we go with that instead, or does it seem clear enough once you have the context of the module loaded up? (I'm happy to defer, here; I do *not* have the context loaded up here, and I'm a big fan of the shortest name possible to get the point across.)

Reply:
It is the package name, yeah. I think it's pretty clear when you have the context, that we're talking about installing a single package, so this is the name. That said, I'll add a comment about how we don't always know the name up-front which should help make it more clear.

Reply:
That makes sense, so I think a comment will close the gap nicely. Thank you!